tests/unittests/test_datasource/test_nocloud.py

```diff
@@ -288,8 +288,25 @@ def test_nocloud_get_devices_freebsd(self, m_is_lxd, fake_blkid):
         self.mocks.enter_context(
             mock.patch.object(util, 'is_FreeBSD', return_value=True))
 
+        def _mfind_devs_with_freebsd(
+                criteria=None, oformat='device',
+                tag=None, no_cache=False, path=None):
+            if not criteria:
+                return ["/dev/msdosfs/foo", "/dev/iso9660/foo"]
+            if criteria.startswith("LABEL="):
+                label = criteria.split("=")[1]
+                if label == "foo":
```

Comment:
Why are we checking label == "foo" here in unit tests? can we drop this label check?